src/astro/sql/operators/sql_dataframe_refactor.py

```diff
@@ -0,0 +1,113 @@
+from typing import Dict
+
+from airflow.decorators.base import DecoratedOperator
+
+from astro.databases import create_database
+from astro.settings import SCHEMA
+from astro.sql.table import Table as OldTable
+from astro.sql.table import TempTable
+from astro.sql.tables import Metadata
+from astro.sql.tables import Table as NewTable
+from astro.utils.dataframe_function_handler import DataframeFunctionHandler
+from astro.utils.table_handler_new import TableHandler
+
+
+class SqlDataframeOperator(DataframeFunctionHandler, DecoratedOperator, TableHandler):
+    def __init__(
+        self,
+        conn_id: str = "",
```

Comment:
I suggest we only support users passing Table objects to this operator without supporting users passing the following arguments: `conn_id, `database`, `schema`, `warehouse` and `role`.